docs/user-guides/devcontainers/working-with-dev-containers.md

```diff
@@ -5,93 +5,159 @@ visual representation of the running environment:
 
 ![Dev container integration in Coder dashboard](../../images/user-guides/devcontainers/devcontainer-agent-ports.png)
 
-## SSH Access
+## SSH access
 
-You can SSH into your dev container directly using the Coder CLI:
+Each dev container has its own agent name, derived from the workspace folder
+(e.g., `/home/coder/my-project` becomes `my-project`). You can find agent names
+in your workspace dashboard, or see
+[Agent naming](./index.md#agent-naming) for details on how names are generated.
+
+### Using the Coder CLI
+
+The simplest way to SSH into a dev container is using `coder ssh` with the
+workspace and agent name:
+
+```console
+coder ssh <workspace>.<agent>
+```
+
+For example, to connect to a dev container with agent name `my-project` in
+workspace `my-workspace`:
+
+```console
+coder ssh my-workspace.my-project
+```
+
+To SSH into the main workspace agent instead of the dev container:
+
+```console
+coder ssh my-workspace
+```
+
+### Using OpenSSH (config-ssh)
+
+You can also use standard OpenSSH tools after generating SSH config entries with
+`coder config-ssh`:
+
+```console
+coder config-ssh
+```
+
+This creates SSH host entries for all your workspaces and their agents,
```

Comment:
Perhaps add a sentence clarifying when (if anytime) the user needs to refresh config-ssh. For instance, when a new container or agent is created.